app/views/layouts/_exp_editor.html.haml

```diff
@@ -7,7 +7,6 @@
   = javascript_tag("ManageIQ.expEditor.second.title = '#{@edit[@expkey][:val2][:title]}';")
 
 #exp_editor_div
-  = render :partial => 'layouts/flash_msg', :locals => {:flash_div_id => 'exp_editor_flash'}
```

Comment:
@d-m-u we need to keep rendering of flash message partial here, to show flash messages specific to expression editor.